src/app/components/form/input/FormInputPassword.vue

```diff
@@ -12,33 +12,57 @@
       <Progress :model-value="strength" class="my-2" />
     </template>
     <template #icon>
-      <IHeroiconsEye v-if="!isVisible" />
-      <IHeroiconsEyeSlash v-else />
+      <IVibetypeClose
```

Comment:
Clicking the icon doesn't clear the input, but I think it should. In the password case, it still hides/unhides the password as the eye-icon currently does.